tests/scripts/test_ray_serve_2.py

```diff
@@ -0,0 +1,26 @@
+import ray
+import sys
+import ray.serve as serve
+import time
+
+def retry_with_timeout(func, timeout=90):
+    err = None
+    for _ in range(timeout):
+        try:
+            return func()
+        except BaseException as e:
+            err = e
+        finally:
+            time.sleep(1)
+    raise err
+
+retry_with_timeout(lambda: ray.init(address='ray://127.0.0.1:10001', namespace=sys.argv[1]))
+retry_with_timeout(lambda: serve.start(detached=True))
+
+@serve.deployment
+def d(*args):
+    return "HelloWorld"
+
+val = retry_with_timeout(lambda: ray.get(d.get_handle().remote()))
```

Comment:
I doubted that whether the serve deployment is the same as the deployment in `test_ray_serve_1.py`. We can test it by updating `return "HelloWorld"` to `return "123"`. If `val` is still equal to `HelloWorld`, maybe the deployment is the same.

Reply:
After I updated return ```HelloWorld``` to return ```123``` in  ```test_ray_serve_1.py```, the ```val``` is still equal to ```HelloWorld```. I will update the test scripts to remove this kind of ambiguity.